greedy_algorithms/jumpgame.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brief Implementation of an algorithm to solve the jumping game problem
+ * @details
+ * This algorithm is a greedy algorithm.
```

Comment:
```suggestion
 * **Problem statement:** Given an array of non-negative integers, you are initially positioned at the first index of the array. Each element in the array represents your maximum jump length at that position. Determine if you are able to reach the last index.
```
This line doesn't add any value to the description.